### PR TITLE
md4qt: add version 2.0.0

### PR DESCRIPTION
**Pull request by igormironchik**
Specify library name and version:  **md4qt/2.0.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

I'm the author of the library.

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/README.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/md4qt/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return "17"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "16",
```
Comment (prince-chrismc) [inline recipes/md4qt/all/conanfile.py:29]:
```suggestion
            "Visual Studio": "16",
            "msvc": "191",
```

File: recipes/md4qt/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        pass
+
+    def package(self):
+        copy(self, "LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "*.hpp", src=os.path.join(self.source_folder, "md4qt"), dst=os.path.join(self.package_folder, "include", "md4qt"))
+
+    def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "md4qt")
+        self.cpp_info.set_property("cmake_target_name", "md4qt::md4qt")
+        self.cpp_info.includedirs.append(os.path.join("include", "md4qt"))
```
Comment (prince-chrismc) [inline recipes/md4qt/all/conanfile.py:68]: 🤔 According to the readme this is not needed? https://github.com/igormironchik/md4qt#example

Though I noticed you are the author so you get a free pass lol

**Review by prince-chrismc — COMMENTED** [review]
2 really tiny nits otherwise this looks super good

**Review by uilianries — APPROVED** [review]
LGTM

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**